integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/index/TestIndexCommand.scala

```diff
@@ -39,7 +39,9 @@ class TestIndexCommand extends QueryTest with BeforeAndAfterAll {
 
   val newClass = "org.apache.spark.sql.CarbonSource"
 
-  test("test index create: don't support using non-exist class") {
+  // TODO: This issue will be fixed during alter change column support for spark 3.1.1
+  // REFER: https://issues.apache.org/jira/browse/CARBONDATA-4210
+  ignore("test index create: don't support using non-exist class") {
```

Comment:
why are these tests commented?

Reply:
changed these test cases back, and changed the expected error message